.github/workflows/platform_tests.yml

```diff
@@ -44,19 +44,19 @@ jobs:
       uses: shogo82148/actions-goveralls@v1
       with:
         path-to-profile: coverage.out
-      if: ${{ runner.os == 'Linux' && matrix.go-version == '1.19' }}
+      if: ${{ runner.os == 'Linux' && matrix.go-version == '1.17' }}
 
     - name: Build WebAssembly binary
       env:
         GOOS: js
         GOARCH: wasm
       working-directory: cmd/fyne_demo
       run: go build
-      if: ${{ matrix.go-version == '1.19' }}
+      if: ${{ matrix.go-version == '1.17' }}
```

Comment:
I would really prefer if wasm was checked against 1.19 as the idea is that wasm come out of latest go and that work, while gopherjs require an older go as backup solution for problematic browser.

Reply:
As stated in my previous comment, I prefer to only have two Go versions at the time. We will slow down our CI times a lot otherwise. Lets just keep this PR simple and fix the issue to make it behave as before